packages/api/models/collections.js

```diff
@@ -169,15 +171,17 @@ class Collection extends Manager {
       ? item : await this.get(item).catch(() => item);
     await this.collectionConfigStore.delete(dataType || name, version);
 
-    const record = {
-      event: 'Delete',
-      deletedAt: Date.now(),
-      record: {
-        name,
-        version
-      }
-    };
-    await publishCollectionSnsMessage(record);
+    if (!process.env.noSNS) {
```

Comment:
I have mixed feelings about using env vars specific to testing in our real API package. See suggestion below